assembly/src/errors.rs

```diff
@@ -104,6 +104,15 @@ pub enum AssemblyError {
         source_file: Option<Arc<SourceFile>>,
         callee: RpoDigest,
     },
+    #[error("invalid exec: exec'd procedures must be available during compilation, but '{callee}' is not")]
```

Comment:
This is not supposed to be an error - `exec` does _not_ require the procedure to be available, and _cannot_ require that.

Reply:
As discussed offline, removed